postgresql_database: Reassign objects owners if database owner changes (#458)
cyrilgdn/terraform-provider-postgresql: website/docs/r/postgresql_database.html.markdown

  force the creation of a new resource as this value can only be changed when a
  database is created.

* `alter_object_ownership` - (Optional) If `true`, the change of the database
  `owner` will also include a reassignment of the ownership of preexisting
  objects like tables or sequences from the previous owner to the new one.
  If set to `false` (the default), then the previous database `owner` will still
  hold the ownership of the objects in that database. To alter existing objects in
  the database, you must be a direct or indirect member of the specified role, or

>   the username in the provider is a superuser.

Reviewer: ```suggestion
  the username in the provider must be superuser.
```
